office/views_admin.py

```diff
@@ -956,10 +967,87 @@ def office_new_view(request):
 
     messages_on_stage = get_messages(request)
     template_values = {
+        'ballotpedia_office_id_dict':              
+        {
+            'label':    'Ballotpedia Office Id (Office Held)',
+            'id':       'ballotpedia_office_id_id',
+            'name':     'ballotpedia_office_id',
+            'value':     ballotpedia_office_id
```

Comment:
please check that I did these values correctly for all of these. my understanding is that since there is a form for a new office contest, we don't need something like `ballotpedia_office_id if ballotpedia_office_id else office_on_stage.ballotpedia_office_id`